hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -448,6 +575,59 @@ public void testRemoveNamespaceProperties() throws TException {
         });
   }
 
+  @Test
+  public void testRemoveNamespaceOwnership() throws TException {
+    Map<String, String> prop = ImmutableMap.of(HiveCatalog.HMS_DB_OWNER, "some_owner");
+    removeNamespaceOwnershipAndVerify("remove_individual_ownership", prop);
+
+    prop =
+        ImmutableMap.of(
+            HiveCatalog.HMS_DB_OWNER,
+            "some_owner",
+            HiveCatalog.HMS_DB_OWNER_TYPE,
+            PrincipalType.GROUP.name());
+    removeNamespaceOwnershipAndVerify("remove_group_ownership", prop);
+
+    prop = ImmutableMap.of();
+
+    removeNamespaceOwnershipAndVerify("remove_ownership_noop", prop);
```

Comment:
This tests that what happens when there is no specific owner/ownertype set for the namespace but then we try to remove them, right?

What would also be beneficial in my opinion is to create the namespace with some owner and then run a noop remove and check if the owner is still the same what we provided during creation.